docs/src/basic-tutorial.md

```diff
@@ -325,16 +324,16 @@ heatmap(randn(20, 20),
 )
 ```
 
-If you set only one attribute, be careful to do `axis = (key = value,)` (note the trailing comma), otherwise you're not making a NamedTuple but a local variable `key`.
+If you set only one attribute, be careful to do `axis = (key = value,)` (note the trailing comma), otherwise you're not creating creating a `NamedTuple` but a local variable `key`.
```

Comment:
```suggestion
If you set only one attribute, be careful to do `axis = (key = value,)` (note the trailing comma), otherwise you're not creating a `NamedTuple` but a local variable `key`.
```